docs/manage/troubleshooting_server_faq.md

```diff
@@ -158,6 +158,59 @@ They will only work once the intake pipeline has run and moved the data into the
 Trips are converted from draft -> processed when the intake pipeline has been run successfully on them.
 - Instructions for running the intake pipeline for a single user (in debug mode): https://github.com/e-mission/e-mission-server#quick-start
 - Instructions for running the intake pipeline as part of a cronjob: https://github.com/e-mission/e-mission-server/wiki/Deploying-your-own-server-to-production#the-analysis-pipeline
+- Examples on how to setup the intake pipeline cronjob inside the docker container are here https://github.com/e-mission/e-mission-docker/tree/master/examples/em-server-multi-tier-cronjob 
+
+If for some motives you need to run the cronjob outside the docker container, here's my little tutorial.
+
+Since we are using source to set the environment for the intake script to run in, we create a little bash script by the name cron_intake.sh:
+```
+#!/bin/bash
+source activate emission
+./e-mission-py.bash bin/intake_multiprocess.py 4
+```
+make sure it's executable:
+```
+chmod a+x cron_intake.sh
+```
+Since we've couldn't figure out how to get the cronjob working inside the docker container, we are going to create the cronjob outside of the container on the host machine. As a first test, we can run this script from outside the container:
```

Comment:
Can you also remove the part about "Since we've couldn't figure out how to get the cronjob working inside the docker container"

Reply:
right :) Done!